code/game/machinery/computer/aifixer.dm

```diff
@@ -1,160 +1,160 @@
-/obj/machinery/computer/aifixer
```

Comment:
line endings got changed here, it was probably on purpose but just pointing it out.

Reply:
I missed this, this is fixed now